src/transforms/onehot.jl

```diff
@@ -47,7 +50,9 @@ function applyfeat(transform::OneHot, feat, prep)
   end
 
   newnms, newcols = first.(onehot), last.(onehot)
-
+  if transform.categorical
+    newcols = [CategoricalArray(new_column, ordered=true) for new_column in newcols]
```

Comment:
We don't use underscore in our code style. Also, the function `categorical(col)` should be used instead of the array constructor.

Reply:
I will work on these reviews and commit the recommened changes 👍
Regarding the ordered set to "true", I found that it is enabling the comparision of elements using operators like "<", ">",
if ordered is set to "false", comparing elements of the array throws an error, which might have been confusing for the user.

Reply:
> Regarding the ordered set to "true", I found that it is enabling the comparision of elements using operators like "<", ">",

I think we don't want to enable these relations with binary variables. A binary variable is a like a yes/no variable with no particular ordering. It is often used for masking or grouping two different subsets of a table.

Reply:
Sure thing 👍
I will keep the output unordered (ordered=false)

Reply:
Thank you @rajatrc1705 , looking forward to review it.